sdk/src/main/java/co/omise/android/models/FlowType.kt

```diff
@@ -12,29 +12,29 @@ import kotlinx.android.parcel.Parceler
  * @see [Sources API](https://www.omise.co/sources-api)
  */
 sealed class FlowType(
-        @JsonValue val name: String?
+        @JsonValue open val name: String?
 ) {
 
     object Redirect : FlowType("redirect")
     object Offline : FlowType("offline")
-    object Unknown : FlowType(null)
+    data class Unknown(override val name: String?) : FlowType(name)
```

Comment:
THIS IS WHAT I WANT!!! Thank you

Reply:
😂😂😂

Reply:
I need it too.

Reply:
Yeah I'm glad you found a way to make it work 👏